main.py

```diff
@@ -329,5 +311,5 @@ async def main():
     opps.update_opportunities_status(internship_data_results)
 
 
-if __name__ == "__main__":
-    asyncio.run(main())
+# if __name__ == "__main__":
+#     asyncio.run(main())
```

Comment:
Why is this commented out? What is `asyncio.run`?

Reply:
This is so the workflow doesn't get triggered. `asyncio.run` has been there since the beginning of this project.

Reply:
> This is so the workflow doesn't get triggered. `asyncio.run` has been there since the beginning of this project.

To temporarily disable the workflow, I’d recommend commenting out the workflow step itself.

Reply:
For sure, will keep that in mind.